ci/infra/testrunner/kubectl/kubectl.py

```diff
@@ -12,11 +12,11 @@ def __init__(self, conf, platform):
         self.platform = platforms.get_platform(conf, platform)
         self.skuba = Skuba(conf, platform)
 
-    def run_kubectl(self, command):
+    def run_kubectl(self, command, stdin=None):
```

Comment:
I love this change. will make tests easier by allowing embedding the manifests in the test as string constant. Thanks